### ISSUE TITLE
Example with petStore not working... Can we have a sample json & example for same ?

### ISSUE THREAD

**Issue by anil614sagar** [apigee-127/swagger-test-templates#85]
```javascript

var stt = require('swagger-test-templates');
var swagger = require('./petStore.json');
var config = {
  assertionFormat: 'should',
  testModule: 'supertest',
  pathName: ['/pet/findByStatus'],
  loadTest: [{pathName:'/pet/findByStatus', operation:'get', load:{requests: 10, concurrent: 1}}, { /* ... */ }],
  maxLen: 80
};

// Generates an array of JavaScript test files following specified configuration
console.log(stt.testGen(swagger, config));
```

Above code generates below output..

[ { name: 'pet-findByStatus-test.js', test: '/pet/findByStatus' },
  { name: '.env',
    test: 'PETSTORE_AUTH=YOUR_TOKEN_GOES_HERE\nAPI_KEY=YOUR_TOKEN_GOES_HERE\n\n' } ]

don't see any test files.. Am i missing something here ?

**Comment by noahdietz** [comment]
This is a bug with the ```pathName``` option. It wasn't passing the test string to the output properly. Opening a PR.

**Comment by noahdietz** [comment]
Also, there was a typo in the example you provided. The path should be ```'/pets/findByStatus'```, there was an 's' missing

**Comment by mm-gmbd** [comment]
Is this now closed?

**Comment by noahdietz** [comment]
fixed with #86
